core/variant/array.cpp

```diff
@@ -563,6 +563,32 @@ bool Array::any(const Callable &p_callable) const {
 	return false;
 }
 
+int Array::find_custom(const Callable &p_callable, const int from) const {
+	const Variant *argptrs[1];
+
+	ERR_FAIL_INDEX_V_MSG(from, size(), -1,
```

Comment:
I think replace this check with an abort when `from` is negative like `find`. `find` and `rfind` don't error on the start position which prevents users from needing to check bounds before calling. If from is past the end, the loop won't execute.